### ISSUE TITLE
Fyrtur blinds spamming log with error

### ISSUE THREAD

**Issue by eriktack** [sanjoyg/dirigera_platform#17]
THe blinds seem to work okay, I can raise and lower them. But they keep spamming the HA log with this, over and over again:
`2024-03-14 09:18:40.052 ERROR (SyncWorker_19) [custom_components.dirigera_platform] error encountered running update on : xxxxxxxxxx
2024-03-14 09:18:40.053 ERROR (SyncWorker_19) [custom_components.dirigera_platform] 6 validation errors for EnvironmentSensor
File "/config/custom_components/dirigera_platform/cover.py", line 126, in update
File "/config/custom_components/dirigera_platform/dirigera_lib_patch.py", line 54, in get_blinds_by_id
File "/usr/local/lib/python3.12/site-packages/dirigera/devices/environment_sensor.py", line 36, in dict_to_environment_sensor
return EnvironmentSensor(dirigeraClient=dirigera_client, **data)
File "/config/custom_components/dirigera_platform/cover.py", line 130, in update
homeassistant.exceptions.HomeAssistantError: (ValidationError(model='EnvironmentSensor', errors=[{'loc': ('attributes', 'currentTemperature'), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('attributes', 'currentRH'), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('attributes', 'currentPM25'), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('attributes', 'maxMeasuredPM25'), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('attributes', 'minMeasuredPM25'), 'msg': 'field required', 'type': 'value_error.missing'}, {'loc': ('attributes', 'vocIndex'), 'msg': 'field required', 'type': 'value_error.missing'}]), 'dirigera_platform', 'hub_exception')`

The instructions to dump json did not work, or I am trying to find the json in the wrong place. What exactly is the "HASS logs"?

**Comment by slajob** [comment]
It looks like you are using an old version of the integration because this bug was fixed in the last release. What's your dirigera_platform version?

**Comment by eriktack** [comment]
Yes, I need to figure out how to enable update notifications for HACS, I dont look in that view that often.

With the newest version 1.3.3 the spam stopped.

All good!
